.github/workflows/build-dev.yml

```
@@ -0,0 +1,19 @@
+name: GitHub Actions - Build
+on:
+  push:
+    branches:
+      - 'dev'
+
+jobs:
+  build-sdk:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+
+      - uses: actions/setup-java@v3
+        with:
+          distribution: 'corretto'
+          java-version: '11'
```

We use java 8. Please, change it for consistency
```suggestion
          java-version: '8'
```